notes/dbIndices.txt

```diff
@@ -14,19 +19,24 @@ db.bestiary_subscriptions.createIndex({"object_id": 1});
 db.bestiary_subscriptions.createIndex({"provider_id": 1});
 
 tomes:
-db.tomes.createIndex({"owner.id": 1});
+db.tomes.createIndex({"owner": 1});  // changed: owner.id -> owner
+
+// to remove
 db.tomes.createIndex({"editors.id": 1});
 db.tomes.createIndex({"subscribers.id": 1});
 db.tomes.createIndex({"active": 1});
 db.tomes.createIndex({"server_active": 1});
 
-compendiums:
-db.compendiums.createIndex({"owner.id": 1});
-db.compendiums.createIndex({"editors.id": 1});
-db.compendiums.createIndex({"subscribers.id": 1});
-db.compendiums.createIndex({"stargazers.id": 1});
-db.compendiums.createIndex({"active": 1});
-db.compendiums.createIndex({"server_active": 1});
+db.tome_subscriptions.createIndex({"type": 1, "subscriber_id": 1});
+db.tome_subscriptions.createIndex({"object_id": 1});
+
+// compendiums:
```

Comment:
Did you mean to leave this commented out?

Reply:
Yup, compendiums haven't been implemented yet (also, this is just a txt file 😛)